common/app/layout/Front.scala

```diff
@@ -476,8 +489,11 @@ object Front extends implicits.Collections {
         (Set.empty[TrailUrl], initialContext, emptyDedupedResultWithPath)
       ) { case ((seenTrails, context, dedupedFrontResult), (pressedCollection, index)) =>
         val omitMPU = pressedPage.metadata.omitMPUsFromContainers(edition)
+        val isNetworkFront = pressedPage.metadata.isNetworkFront
         val container = Container.fromPressedCollection(pressedCollection, omitMPU)
-        val (newSeen, newItems, (usedAndDeduped, usedButNotDeduped)) = deduplicate(seenTrails, container, pressedCollection.curatedPlusBackfillDeduplicated)
+        val isEditorialFront = ConfigAgent.getPriorityForPath(pressedPage.id).getOrElse("editorial") == "editorial"
```

Comment:
This seems slightly odd... The default behaviour is to say yes it is `editorial`? Maybe that's right.
I would have expected to see `== Some("editorial")`, but that would mean it wouldn't default to editorial.

There would be something seriously wrong if the `ConfigAgent` didn't contain the `pressedPage.id` though.